plugins/linuxv2/model/l3/l3.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package l3;
+
+/* static ip route */
+message LinuxStaticRoute {
+    string outgoing_interface = 1; /* outgoing interface logical name (mandatory) */
+
+    enum Scope {
+        GLOBAL = 0;
+        SITE = 1;
+        LINK = 2;
+        HOST = 3;
+    }
+    Scope scope = 2;               /* the scope of the area where the link is valid */
+
```

Comment:
I don't know if it happened by accident, but it seems you missed some of the fields for static routes that seemed relevant, like: `src_ip_addr` `table` (VRF) and `namespace`.